src/converters/fromZigbee.ts

```diff
@@ -728,6 +728,36 @@ const converters1 = {
             return payload;
         },
     } satisfies Fz.Converter,
+    seMetering: {
```

Comment:
could we also merge this with https://github.com/Koenkk/zigbee-herdsman-converters/blob/4a18e2966318e3153cff32198c71dc554528d72a/src/converters/fromZigbee.ts#L697 ?